credential-schema.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@mtuchi , this should follow the JSON schema format that you and Elias were using for lightning.

Reply:
Resolved https://github.com/OpenFn/language-fhir/pull/3/commits/03e4ae27f0954a61e51ab43e5f36b65a8e795f43